README.md

```diff
@@ -231,6 +231,16 @@ E.g. InTheFuture => `where customer0_.date_of_next_special_offer > localtimestam
 
 Usage: `@Spec(path="dateOfTheNextOffer", spec=InTheFuture.class)`.
 
+### EqualDay ###
+
+Supports date-type fields. Filters the results that are within the particular date (day). Internally uses `Between` specification with range within one day from `00:00:00` to `23:59:59`. When datetime format with a time part will be specified and provided, then the time part will be ignored. For example, providing `2022-12-20T08:45:57` parameter for `LocalDateTime` field will result with filtering with range from `2022-12-20T00:00:00` to `2022-12-20T23:59:59`.  
```

Comment:
Filters the results that are within the particular date (day). ---> Matches the day part of a date-time attribute, ignoring the time.